front/components/items/Appointment/helper.ts

```diff
@@ -28,3 +28,22 @@ export const getDateOfAppointments = ({ appointments }: props) => {
         }
     });
 };
+
+export const getIdOfAppointment = ({
+    fullDataAppointment,
+    chosenTherapist,
+    chosenDate,
+    chosenTime,
+}: propsOfGetId) => {
+    return fullDataAppointment?.filter((item) => {
+        if (item.attributes.therapist.data.id === chosenTherapist) {
+            if (chosenDate !== undefined) {
```

Comment:
```suggestion
            if (chosenDate) {
```
is enough if you specificly dont want to pass values like {null, 0, false}